src/haxeLanguageServer/features/haxe/codeAction/DiagnosticsCodeActionFeature.hx

```diff
@@ -483,4 +499,51 @@ class DiagnosticsCodeActionFeature implements CodeActionContributor {
 		}
 		return actions;
 	}
+
+	function getNewVariablePosition(document:HaxeDocument, classToken:TokenTree, fieldScope:JsonClassFieldScope):Null<Range> {
+		final brOpen = classToken!.getFirstChild()!.getFirstChild();
```

Comment:
finding `{` by only looking at firstChilds will fail in case of ` extends`, `implements` or metadata.
better way would be to use 
`classToken.access().firstChild().firstOf(BrOpen).token` (assuming `using tokentree.TokenTreeAccessHelper`)